src/app/forms/templates/test-records/master.template.ts

```diff
@@ -475,6 +476,15 @@ export const masterTpl: Record<VehicleTypes, Partial<Record<keyof typeof TEST_TY
       notes: NotesSection,
       reasonForCreation: reasonForCreationSection
     },
+    testTypesDeskBasedGroup3: {
```

Comment:
This is in the master template, which is for amending tests.
Sections for creating tests go in the create master template.

Reply:
Thanks for pointing that out. I have not done any tests in a long while so I was not aware of the difference. I will address this.

Reply:
This is now moved into the create master template.